library/CHART-TEMPLATE/templates/example-cm-staging.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.naming.fullname" . }}-examples-staging
+  namespace: {{ .Release.Namespace }}
+data:
+  {{- /* Be careful because this returns an empty string when false because "false" is evaluated as true in Helm
+         Take a look to the tests suite "common_library_lowdatamode_test.yaml" to see who this behaves */}}
```

Comment:
is this comment correct or you meant the new file?

Reply:
Nope, it was a typo during copy and pasting from `lowDataMode` because I would like to have the same code style in all the helpers by copying and adapting them.

Solved in [#25066d2](https://github.com/newrelic/helm-charts/pull/755/commits/25066d2246476e48e0c02c0d83751c6b52f359ce)